test_scripts/WebEngine/WebEngineProjection/025_Check_WSS_connection_for_WebEngine_projection_app_in_NONE.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that SDL doesn't close the connection with WebEngine
+-- projection app if the app was closed (HMILevel NONE is assigned)
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+
+-- Sequence:
+-- 1. Application registers with WEB_VIEW appHMIType
+--  a. SDL successfully registers application (resultCode SUCCESS, success:"true")
+--  b. SDL assigns HMILevel (NONE) to the WebEngine projection app and doesn't close the WebSocket connection
+-- 2. Activate web application
+--  a. WebEngine projection application successfully activated using remained connection to SDL
+-- 3. Deactivate web application to NONE and check connection
+--  a. Exit from application (reason = "USER_EXIT")
+-- 4. Check WebSocket connection
```

Comment:
I would propose to clarify step for ex. 
```suggestion
-- 4. Activate web application (Check WebSocket connection)
```